packages/components/src/toolbar/stories/index.tsx

```diff
@@ -106,6 +114,7 @@ Default.args = {
 				<ToolbarButton icon={ link } label="Link" />
 				<ToolbarGroup
 					isCollapsed
+					// @ts-ignore TODO: Remove when ToolbarGroup is typed
```

Comment:
We generally prefer `@ts-expect-error` over a simple `@ts-ignore` so it's flagged when it's no longer an error. It helps prevent any outdated ignore statements from accumulating.

```suggestion
					// @ts-expect-error TODO: Remove when ToolbarGroup is typed
```

Reply:
Got it, thanks for the tip! I will replace ignores with `@ts-expect-error` wherever possible.